packages/botonic-plugin-google-analytics/src/index.js

```diff
@@ -10,8 +10,8 @@ export default class BotonicPluginGoogleAnalytics {
    * @param {boolean} [options.trackManually] - If set to true, no default tracking will be done in post method.
    */
   constructor(options) {
-    this.userId = options.userId ?? this.defaultUserId
-    this.userTraits = options.userTraits ?? this.defaultUserTraits
+    this.userId = options.userId ?? this.getUserId
```

Comment:
sorry I meant to rename options.userId to options.getUserId so that it's clear for the developer that it's a function. what do you think?
about the defaultUserId, defaultUserTraits functions, since the developers don't need to interact with them, their name is not so important. defaultGetUserId or getUserIdFromSession sound good to me

Reply:
Oh! ok, now I understand and I do like the rename, it's clearer. Done! Thanks!